### PR TITLE
Added: when JAVA_HOME environment variable is not found,trigger

### PR DESCRIPTION
**Pull request by lnsane**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

A friendly prompt when the JAVA_HOME environment variable cannot be found

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: distribution/bin/startup.sh [hunk 1]
```diff
@@ -47,7 +54,12 @@ if [ -z "$JAVA_HOME" ]; then
     fi
   fi
   if [ -z "$JAVA_HOME" ]; then
-        error_exit "Please set the JAVA_HOME variable in your environment, We need java(x64)! jdk8 or later is better!"
+    if [ ! -d "$(dirname $(readlink -f $0))/jdk8u262-b10" ]; then
```
Comment (KomachiSion) [inline distribution/bin/startup.sh:57]: hint command locks the version of jdk `jdk8u262-b10` may be not good.

**Comment by KomachiSion** [comment]
@lnsane Please sign the SLA

File: distribution/bin/startup.sh (hunk 1 above)
Comment (lnsane) [reply distribution/bin/startup.sh:57]: > hint command locks the version of jdk `jdk8u262-b10` may be not good.

It does not look good, I just have this idea, from my point of view: the download address will not change, so the file name will not change.

Comment (KomachiSion) [reply distribution/bin/startup.sh:57]: I think the hint is hard to do. 

Download the official JDK8 need oracle account. 
Use github or openJDK need bind the version is not very good.

So I prefer to keep it as now. Tell users nacos config need JAVA_HOME, let users choose there own version and install to users' path.

**Pull request closed without merging**